src/stdio.jl

```diff
@@ -29,17 +59,37 @@ function send_stream(rd::IO, name::AbstractString)
             # FIXME: what should we do here?
             string("<ERROR: invalid UTF8 data ", d, ">")
         end
-        send_ipython(publish,
-                     msg_pub(execute_msg, "stream",
-                             @compat Dict("name" => name, "text" => s)))
+    else
+        s = ""
     end
+    s, nb
```

Comment:
It seems like this should just return `d` now.  There's no reason to convert it into a string if we are just going to write the bytes to a buffer in `watch_stream`